src/App.tsx

```diff
@@ -1,7 +1,33 @@
 import React from 'react';
 
-export const App: React.FC = () => (
-  <div className="App">
-    <p className="App__message">The last pressed key is [Enter]</p>
-  </div>
-);
+interface State {
+  lastKey: string,
+}
+
+export class App extends React.Component<{}, State> {
+  state: State = {
+    lastKey: 'Nothing was pressed yet',
```

Comment:
The initial state should be null. The state name is lastKey but `'Nothing was pressed yet'` doesn't look like a key does it?
```suggestion
    lastKey: null,
```